Файл squashfs-root.6.mod/dev/console это специальный файл с посимвольным доступом, тогда как файл squashfs-root.7.mod/dev/console ? специальный файл с посимвольным доступом
diff -ur squashfs-root.6.mod/etc/config/server_mapping squashfs-root.7.mod/etc/config/server_mapping
--- squashfs-root.6.mod/etc/config/server_mapping	2020-08-06 14:21:37.000000000 +0500
+++ squashfs-root.7.mod/etc/config/server_mapping	2021-11-27 22:34:46.917037163 +0500
@@ -13,32 +13,32 @@
 	option S s.miwifi.com
 	option APP app.miwifi.com
 	option STUN in.stun.miwifi.com
-	option BROKER in.broker.miwifi.com
+	option BROKER broker.miwifi.com
 
 config server 'server_EU'
 	option API eu.api.miwifi.com
 	option S s.miwifi.com
 	option APP app.miwifi.com
 	option STUN eu.stun.miwifi.com
-	option BROKER eu.broker.miwifi.com
+	option BROKER broker.miwifi.com
 
 config server 'server_RU'
 	option API ru.api.miwifi.com
 	option S s.miwifi.com
 	option APP app.miwifi.com
 	option STUN ru.stun.miwifi.com
-	option BROKER ru.broker.miwifi.com
+	option BROKER broker.miwifi.com
 
 config server 'server_SG'
 	option API sg.api.miwifi.com
 	option S s.miwifi.com
 	option APP app.miwifi.com
 	option STUN sg.stun.miwifi.com
-	option BROKER sg.broker.miwifi.com
+	option BROKER broker.miwifi.com
 
 config server 'server_US'
 	option API us.api.miwifi.com
 	option S s.miwifi.com
 	option APP app.miwifi.com
 	option STUN us.stun.miwifi.com
-	option BROKER us.broker.miwifi.com
+	option BROKER broker.miwifi.com
diff -ur squashfs-root.6.mod/usr/lib/lua/luci/view/web/apsetting/wifi.htm squashfs-root.7.mod/usr/lib/lua/luci/view/web/apsetting/wifi.htm
--- squashfs-root.6.mod/usr/lib/lua/luci/view/web/apsetting/wifi.htm	2021-03-12 17:48:10.000000000 +0500
+++ squashfs-root.7.mod/usr/lib/lua/luci/view/web/apsetting/wifi.htm	2021-11-28 01:00:43.103203707 +0500
@@ -91,6 +91,30 @@
             </div>
         </div>
 
+        <div class="mod-set mod-wifi">
+            <div class="hd">
+                <h3>WPS Switching</h3>
+                <div class="switch">
+                    <a href="#WPSswitch" id="WPSswitch" class="btn-switch btn-switch-on" data-on="1"></a>
+                </div>
+            </div>
+            <div class="bd">
+                <p>Enabling and disabling the WPS</p>
+            </div>
+        </div>
+
+        <div class="mod-set mod-wifi">
+            <div class="hd">
+                <h3>LED Switching</h3>
+                <div class="switch">
+                    <a href="#LEDswitch" id="LEDswitch" class="btn-switch btn-switch-on" data-on="1"></a>
+                </div>
+            </div>
+            <div class="bd">
+                <p>Enabling and disabling the LED indicators</p>
+            </div>
+        </div>
+
 
     </div>
 
@@ -403,7 +427,43 @@
 
         });
 
+        //get wps info
+         $.getJSON('<%=luci.dispatcher.build_url("api","xqsystem","wps_status")%>')
+        .done(function( rsp ){
+            if ( rsp.code == 0 ) {
+                 if( rsp.status == 1 || rsp.status == 2 ){
+                    $('#WPSswitch')
+                        .removeClass('btn-switch-off')
+                        .addClass('btn-switch-on')
+                        .attr('data-on', '1');
+                 }else{
+                    $('#WPSswitch')
+                        .removeClass('btn-switch-on')
+                        .addClass('btn-switch-off')
+                        .attr('data-on', '0');
+                 }
+            }
 
+        });
+
+        //get led info
+         $.getJSON('<%=luci.dispatcher.build_url("api","misystem","led")%>')
+        .done(function( rsp ){
+            if ( rsp.code == 0 ) {
+                 if( rsp.status == 1 ){
+                    $('#LEDswitch')
+                        .removeClass('btn-switch-off')
+                        .addClass('btn-switch-on')
+                        .attr('data-on', '1');
+                 }else{
+                    $('#LEDswitch')
+                        .removeClass('btn-switch-on')
+                        .addClass('btn-switch-off')
+                        .attr('data-on', '0');
+                 }
+            }
+
+        });
 
         // get wifi and rander html
         var containers = ['wifiset24', 'wifiset50', 'wifisetguest'],
@@ -856,8 +916,90 @@
 
         });
 
+        $('#WPSswitch').on('click', function(e){
+            e.preventDefault();
+            var on = $(this).attr('data-on') == 1 ? 0:1;
+            var requestURL = on == 1 ? '<%=luci.dispatcher.build_url("api", "xqsystem", "wps")%>' : '<%=luci.dispatcher.build_url("api", "xqsystem", "wps_cancel")%>';
+            var self = this;
+            var requestData = {
+                on: on
+            };
+            $.ajax({
+                url: requestURL,
+                dataType: 'json',
+                timeout: 5000,
+                type: 'POST',
+                data: requestData,
+                success: function( rsp ){
+                    if( rsp.code === 0 ){
+                        if(on == 0) {
+                            $(self)
+                            .removeClass('btn-switch-on')
+                            .addClass('btn-switch-off')
+                            .attr('data-on', '0');
+                        }else {
+                            $(self)
+                            .removeClass('btn-switch-off')
+                            .addClass('btn-switch-on')
+                            .attr('data-on', '1');
+                        }
+                    } else {
+                        if ( rsp.code !== 401) {
+                            var msg = StringH.encode4Html( rsp.msg );
+                            $.alert( msg ).lock();
+                        }
+                    }
+                },
+                error: function() {
+                    $.alert( '<%:WPS switching failed%>' );
+                }
+            });
+
+        });
+
+        $('#LEDswitch').on('click', function(e){
+            e.preventDefault();
+            var on = $(this).attr('data-on') == 1 ? 0:1;
+            var self = this;
+            var requestData = {
+                on: on
+            };
+            $.ajax({
+                url: '<%=luci.dispatcher.build_url("api", "misystem", "led")%>',
+                dataType: 'json',
+                timeout: 5000,
+                type: 'POST',
+                data: requestData,
+                success: function( rsp ){
+                    if( rsp.code === 0 ){
+                        if(on == 0) {
+                            $(self)
+                            .removeClass('btn-switch-on')
+                            .addClass('btn-switch-off')
+                            .attr('data-on', '0');
+                        }else {
+                            $(self)
+                            .removeClass('btn-switch-off')
+                            .addClass('btn-switch-on')
+                            .attr('data-on', '1');
+                        }
+                    } else {
+                        if ( rsp.code !== 401) {
+                            var msg = StringH.encode4Html( rsp.msg );
+                            $.alert( msg ).lock();
+                        }
+                    }
+                },
+                error: function() {
+                    $.alert( '<%:LED switching failed%>' );
+                }
+            });
+
+        });
+
+
+
 
-          
         $('#bsdswitch').on('click', function(e){
             e.preventDefault();
             var on = $(this).attr('data-on');
diff -ur squashfs-root.6.mod/usr/lib/lua/luci/view/web/setting/wifi.htm squashfs-root.7.mod/usr/lib/lua/luci/view/web/setting/wifi.htm
--- squashfs-root.6.mod/usr/lib/lua/luci/view/web/setting/wifi.htm	2021-03-18 14:55:45.024183937 +0500
+++ squashfs-root.7.mod/usr/lib/lua/luci/view/web/setting/wifi.htm	2021-11-28 01:00:46.299235487 +0500
@@ -132,6 +132,18 @@
 
         <div class="mod-set mod-wifi">
             <div class="hd">
+                <h3>WPS Switching</h3>
+                <div class="switch">
+                    <a href="#WPSswitch" id="WPSswitch" class="btn-switch btn-switch-on" data-on="1"></a>
+                </div>
+            </div>
+            <div class="bd">
+                <p>Enabling and disabling the WPS</p>
+            </div>
+        </div>
+
+        <div class="mod-set mod-wifi">
+            <div class="hd">
                 <h3>LED Switching</h3>
                 <div class="switch">
                     <a href="#LEDswitch" id="LEDswitch" class="btn-switch btn-switch-on" data-on="1"></a>
@@ -455,6 +467,25 @@
 
         });
 
+        //get wps info
+         $.getJSON('<%=luci.dispatcher.build_url("api","xqsystem","wps_status")%>')
+        .done(function( rsp ){
+            if ( rsp.code == 0 ) {
+                 if( rsp.status == 1 || rsp.status == 2 ){
+                    $('#WPSswitch')
+                        .removeClass('btn-switch-off')
+                        .addClass('btn-switch-on')
+                        .attr('data-on', '1');
+                 }else{
+                    $('#WPSswitch')
+                        .removeClass('btn-switch-on')
+                        .addClass('btn-switch-off')
+                        .attr('data-on', '0');
+                 }
+            }
+
+        });
+
         //get led info
          $.getJSON('<%=luci.dispatcher.build_url("api","misystem","led")%>')
         .done(function( rsp ){
@@ -964,6 +995,47 @@
                 }
             });
 
+        });
+
+        $('#WPSswitch').on('click', function(e){
+            e.preventDefault();
+            var on = $(this).attr('data-on') == 1 ? 0:1;
+            var requestURL = on == 1 ? '<%=luci.dispatcher.build_url("api", "xqsystem", "wps")%>' : '<%=luci.dispatcher.build_url("api", "xqsystem", "wps_cancel")%>';
+            var self = this;
+            var requestData = {
+                on: on
+            };
+            $.ajax({
+                url: requestURL,
+                dataType: 'json',
+                timeout: 5000,
+                type: 'POST',
+                data: requestData,
+                success: function( rsp ){
+                    if( rsp.code === 0 ){
+                        if(on == 0) {
+                            $(self)
+                            .removeClass('btn-switch-on')
+                            .addClass('btn-switch-off')
+                            .attr('data-on', '0');
+                        }else {
+                            $(self)
+                            .removeClass('btn-switch-off')
+                            .addClass('btn-switch-on')
+                            .attr('data-on', '1');
+                        }
+                    } else {
+                        if ( rsp.code !== 401) {
+                            var msg = StringH.encode4Html( rsp.msg );
+                            $.alert( msg ).lock();
+                        }
+                    }
+                },
+                error: function() {
+                    $.alert( '<%:WPS switching failed%>' );
+                }
+            });
+
         });
 
         $('#LEDswitch').on('click', function(e){
diff -ur squashfs-root.6.mod/usr/share/xiaoqiang/xiaoqiang_version squashfs-root.7.mod/usr/share/xiaoqiang/xiaoqiang_version
--- squashfs-root.6.mod/usr/share/xiaoqiang/xiaoqiang_version	2021-03-18 15:43:12.075104493 +0500
+++ squashfs-root.7.mod/usr/share/xiaoqiang/xiaoqiang_version	2021-11-27 22:36:21.964266776 +0500
@@ -1,6 +1,6 @@
 config core 'version'
 	# ROM ver
-	option ROM '3.0.36_mod6'
+	option ROM '3.0.36_mod7'
 	# channel
 	option CHANNEL 'release'
 	# hardware platform
